packages/web3-validator/src/utils.ts

```diff
@@ -450,6 +450,7 @@ export function uint8ArrayToHexString(uint8Array: Uint8Array): string {
 }
 
 // for optimized technique for hex to bytes conversion
+// many thanks to @noble/hashes and viem from where this implementation is taken <3
```

Comment:
A tiny suggestion to count for the tiny enhancements mentioned at: https://github.com/web3/web3.js/pull/6596#pullrequestreview-1736321024 :smile:. If you like to do so. 
```suggestion
// many thanks to @noble/hashes and viem from where this implementation is taken <3 and then enhanced a bit
```